pwiz_tools/Skyline/Controls/Graphs/AreaReplicateGraphPane.cs

```diff
@@ -320,8 +323,11 @@ public override void UpdateGraph(bool selectionChanged)
                         if (parentGroupNode.HasLibInfo)
                             ExpectedVisible = AreaExpectedValue.library;
                     }
+                    if (ratioToLabel != null && !ratioToLabel.IsotopeLabelType.IsLight && isShowingMsMs)

```

Comment:
Why is the "!ratioToLabel.IsotopeLabelType.IsLight" here? I would expect that the user could have their normalization either be "ratio to heavy" or "ratio to light", and everything would mostly behave the same but reversed.